web/client/components/widgets/enhancers/tools/pinnableWidget.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018, GeoSolutions Sas.
```

Comment:
2019 :blush:

Reply:
Some of these files has been created in 2018 :smile: